docs(NextJS): use auth method basic for guides (#4802)
zitadel/zitadel: docs/docs/examples/login/nextjs-b2b.md

Then on the project detail page click on new application and enter a name for this app.
Let's call this one `portal-web`.
Select `Web`, continue, `CODE`, then enter `http://localhost:3000/api/auth/callback/zitadel` for the redirect, and `http://localhost:3000` for the post redirect. Then press on `create`.

> As the requests from your application to ZITADEL are made on NextJS serverside, you can safely select `CODE`. This makes sure you still get a secret which is then used in combination with PKCE. Note that the secret never gets exposed on the browser and is kept in a confidential environment.

Reviewer: ```suggestion
Because the requests from your NextJS application to ZITADEL are made on the server side, you can safely select `CODE`. With this you still get a secret which is then usable alongside PKCE. Your secret never gets exposed on the browser since it is kept in your NextJS server.
```

Reviewer: I did slightly changes to for SEO purpose to reduce the amount of redundant sentences.
